game_patch/multi/network.cpp

```diff
@@ -620,6 +620,17 @@ struct df_sign_packet_ext
     uint8_t version_minor = VERSION_MINOR;
 };
 
+struct af_join_req_packet_ext_v2
```

Comment:
Both structures have the same beginning (one is part of the other). Did Goober add fields at the end without increasing the version? How the server know what fields are available?

Reply:
Ah, makes sense, I don't remember if I tested this code with Alpine 1.1 or 1.0, but I guess I used the format from AF 1.0